index.html

```diff
@@ -2049,9 +2049,10 @@ <h3 id="Schema-Building">Schema Building:</h3>
     </p>
 
     <p id="Schema-foreign">
-      <b class="header">foreign</b><code>table.foreign(column))</code>
+      <b class="header">foreign</b><code>table.foreign(~mixed~)</code>
```

Comment:
Do you want to keep the remaining doc intact? It has both calls
demonstrated.

On Tue, Mar 8, 2016 at 9:58 AM, wubzz <notifications@github.com> wrote:

> In index.html
> <https://github.com/tgriesser/knex/pull/1261#discussion_r55369744>:
>
> > @@ -2049,9 +2049,10 @@ <h3 id="Schema-Building">Schema Building:</h3>
> >      </p>
> >
> >      <p id="Schema-foreign">
> > -      <b class="header">foreign</b><code>table.foreign(column))</code>
> > +      <b class="header">foreign</b><code>table.foreign(~mixed~)</code>
>
> Could we change this to plural columns instead of ~mixed~ ? I know it
> doesn't make much sense seeing as it accepts both Array and Non-Array
> string literals, but I ask this to keep the consistency of the
> documentation as much as possible.
>
> IE, .dropForeign right below states columns
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgriesser/knex/pull/1261/files#r55369744>.
>